### PR TITLE
Fixes #226: Added licenses methods

### LINKED ISSUE #226: Add Licenses methods
See also http://docs.gitlab.com/ce/api/licenses.html
- `licenses`: List available licenses.
  - http://docs.gitlab.com/ce/api/licenses.html#list-license-templates
  - `GET /licenses`
- `get_license`: Get a specific license and its information.
  - http://docs.gitlab.com/ce/api/licenses.html#single-license-template
  - `GET /licenses/:key`

### PR DESCRIPTION
**Pull request by alexives**

### CODE REVIEW

**Comment by asedge** [comment]
A minor nitpick but any reason you chose `Licenses` rather than `License`?  It's named `License` on the API docs: https://docs.gitlab.com/ee/api/license.html#license

**Comment by alexives** [comment]
I will fix that!

File: lib/gitlab/client/licenses.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+class Gitlab::Client
+  # Defines methods related to licenses.
+  # @see https://docs.gitlab.com/ce/api/licenses.html
```
Comment (NARKOZ) [inline lib/gitlab/client/licenses.rb]: the URL returns 404

File: lib/gitlab/client/licenses.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  module Licenses
```
Comment (NARKOZ) [inline lib/gitlab/client/licenses.rb]: Let's namespace it as `Templates`. It makes more sense considering http://docs.gitlab.com/ce/api/templates/gitignores.html and http://docs.gitlab.com/ce/api/templates/gitlab_ci_ymls.html

**Comment by NARKOZ** [comment]
ping @alexives

**Comment by NARKOZ** [comment]
Closing since there was no response.

**Pull request closed without merging**